addons/website_sale/static/src/scss/product_configurator.scss

```diff
@@ -1,31 +1,26 @@
 .css_attribute_color {
+    --css-attribute-color-border-width: .25rem;
+
     position: relative;
     display: inline-block;
-    border: 5px solid $input-border-color;
+    border: var(--css-attribute-color-border-width) solid $body-bg;
+    width: calc(2rem - var(--css-attribute-color-border-width));
+    height: calc(2rem - var(--css-attribute-color-border-width));
```

Comment:
We should expose the "initial" height (2rem) using CSS variables as well ... it would be interesting to customize.